docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -402,7 +427,9 @@ def my_slack_on_run_success(context: RunStatusSensorContext):
     )
 ```
 
-Then, you can add the sensor to a repository so it can be enabled and used the same as other sensors:
+When you write a run status sensor that doesn't return anything, an event is added to the run's event log to indicate that the sensor ran.
```

Comment:
```suggestion
When a run status sensor is triggered by a job run but doesn't return anything, Dagster will report an event back to the run to indicate that the sensor ran.
```

as im writing this out and per discussion with @prha offline, i find the behavior of run status sensor could expand to be confusing / hard to explain, and i'm now more convinced that we may eventually want to separate 1) run status sensor that acts on run status to start another run, vs 2) a job-level hook that alerts external services on failure/success. 

i think this PR pushes us towards 1) a lot more, while this my_slack_on_run_success is the 2) case. the notes and examples here should be good to address the confusion now, but eventually i feel we may want to move 2) to a separate surface area (other than sensor).